docs/_quarto.yml

```diff
@@ -367,9 +367,63 @@ quartodoc:
             name: Temporal expressions
             desc: Dates, times, timestamps and intervals.
           contents:
-            - TimestampValue
-            - DateValue
-            - TimeValue
+            - name: TimestampValue
+              include_inherited: true
```

Comment:
```suggestion
```
See Phillip comment below, I tested this locally and we don't need this if we list the members